core/src/main/java/io/aiven/klaw/service/TopicSyncControllerService.java

```diff
@@ -1056,6 +1056,7 @@ public ApiResponse updateSyncTopics(List<SyncTopicUpdates> updatedSyncTopics)
     }
 
     if (updatedSyncTopics.size() == 0 && updatedSyncTopicsDelete.size() > 0) {
+      manageDatabase.loadTopicsForOneTenant(tenantId);
       return ApiResponse.builder().success(true).message(ApiResultStatus.SUCCESS.value).build();
```

Comment:
This is good we should do a check next week in a bit of free time on if there are any other paths we need to check for reloading topics and other cached info.